application/src/main/resources/thingsboard.yml

```diff
@@ -1693,6 +1693,9 @@ queue:
       print-interval-ms: "${TB_QUEUE_KAFKA_CONSUMER_STATS_MIN_PRINT_INTERVAL_MS:60000}"
       # Time to wait for the stats-loading requests to Kafka to finish
       kafka-response-timeout-ms: "${TB_QUEUE_KAFKA_CONSUMER_STATS_RESPONSE_TIMEOUT_MS:1000}"
+    lag-report:
+      # Interval for reporting downlink lag per Edge.
```

Comment:
Let's move this to edges: ?
so, it's going to be 
edges.kafka.lag-report-to-ts.enabled
edges.kafka.lag-report-to-ts.send-ts-interval-ms